### PR TITLE
Enable RSS feed

### PR DESCRIPTION
**Pull request by wezm**

### CODE REVIEW

**Comment by jplatte** [comment]
Thanks for reminding me, I did want to enable this. I think there's more to be done than just that option though.

**Comment by wezm** [comment]
Yep there's the `link` tag as well. I added a commit with it included.

File: templates/skel.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <meta charset="utf-8">
 <meta http-equiv="x-ua-compatible" content="ie=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<link rel="alternate" type="application/rss+xml" href="{{ config.base_url }}/rss.xml" title="blog.turbo.fish" />
```
Comment (jplatte) [inline templates/skel.html]: I generally don't use `config.base_url` to make local testing easier. Instead I just use site-relative links.

```suggestion
<link rel="alternate" type="application/rss+xml" href="/rss.xml" title="blog.turbo.fish" />
```

**Comment by jplatte** [comment]
I still want to also link to the feed directly and potentially customize the RSS, but I guess this can be merged before any of that happens.

**Comment by jplatte** [comment]
I have no idea why GitHub's "Squash and merge" set me as the author of the squash commit, it doesn't normally do that.

**Comment by jplatte** [comment]
Ah, it didn't squash at all. Fixed by doing the squashing manually and force-pushing.

**Comment by jplatte** [comment]
Well, that doesn't seem like it worked particularily well. At least my feed reader can't process that file. I can't look into it right now though.

**Comment by wezm** [comment]
W3C feed validator says it's valid: https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fblog.turbo.fish%2Frss.xml

**Comment by jplatte** [comment]
Well, that doesn't really buy me anything. Also, the feed has an empty title, which isn't very nice. Either way, I don't have the time to fix any of that right now, but have raised https://github.com/getzola/zola/issues/967 for the first issue I've encountered with the default-generated feed.

**Comment by wezm** [comment]
> Well, that doesn't really buy me anything.

Sorry, just trying to help out as having an RSS feed makes it easier for me to follow your blog and publish posts to [Read Rust](https://readrust.net/).

**Comment by jplatte** [comment]
Of course. Did it work for you at least?

**Comment by wezm** [comment]
Yep, I have subscribed.

**Pull request merged**